examples/markdown/src/bin/example_markdown_app.rs

```diff
@@ -22,7 +22,8 @@ fn main() {
         .flatten()
         .expect_throw("failed to query root element");
 
-    let mut bridge = MarkdownWorker::spawner().spawn("/example_markdown_worker.js");
+    let mut bridge =
+        MarkdownWorker::spawner().spawn_with_loader("/example_markdown_worker_loader.js");
```

Comment:
Wonder if trunk can set a build-time env so this path doesn't need to be hard coded